bosh-init/bosh_init_cache/minimal-state.json

```diff
@@ -0,0 +1,3 @@
+{
+  "installation_id": "44f01911-a47a-4a24-6ca3-a3109b33f058"
```

Comment:
Can you add to the commit message how this was generated? Is it significant?

I thought it might be a [V4 (random) UUID](https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_.28random.29) but it doesn't seem to be.